### ISSUE TITLE
FG - Action tab is not working 

### ISSUE THREAD

**Issue by sumshri** [rusty1909/Navigator#43]
Hi Team ,

Basic information tab is working on Company details page . But Action tab is not working .
I am not able delete and edit company Employee  .
Please do the needful .

Thank u

**Comment by rusty1909** [comment]
company/action.php

```
case "delete" :
    $id = $_GET['id'];
    $mUser = new User($id);

    if($mUser->delete())
        header('Location:index.php');
    else {
        header('Location:index.php');
    }

    break;
```

problem is in above code,
please check if $mUser is created properly

@dheerajagrawal19

**Comment by rusty1909** [comment]
fixed by commit # 8e91875
